ci/build_container/build_recipes/googletest.sh

```diff
@@ -1,14 +0,0 @@
-#!/bin/bash
-
-set -e
-
-VERSION=release-1.8.0
```

Comment:
I know that @jmillikin-stripe is doing work here, but can we stick with the existing prebuilt scripts for now? You can checkout the code directly from git and keep the rest of everything the same. See https://github.com/envoyproxy/envoy/blob/master/ci/build_container/build_recipes/boringssl.sh for an example of how this is done w/ Git. You will want to do a commit with just the switch to git/SHA for googletest. Then that will make a new build image. Then after that you can use that image SHA for this PR.

Reply:
You don't even need to switch to using Git checkouts. Change this single line to:

`VERSION=43863938377a9ea1399c0596269e0890b5c5515a`

Reply:
Neat trick I did not know that. Or that. :)

Reply:
oh, that would be fantastic as I think it would do minimal damage to our internal merges.

I'll send out a separate PR just for the hash change, then once docker updates I'll roll back the parts of this I can get rid of.